plugins/inputs/cloud_pubsub/pubsub.go

```diff
@@ -44,7 +44,8 @@ type PubSub struct {
 
 	Base64Data bool `toml:"base64_data"`
 
-	Log telegraf.Logger
+	Log             telegraf.Logger
+	ContentEncoding string `toml:"content_encoding"`
 
```

Comment:
Will this have / does it need a default value?

Reply:
I think empty string is the default. https://github.com/influxdata/telegraf/blob/master/internal/content_coding.go#L17

Reply:
Cool. gotcha.